### PR TITLE
Show error

### PR DESCRIPTION
**Pull request by 13guff13**

### CODE REVIEW

**Comment by jkanche** [comment]
@13guff13 

why not use `on-error` handler on `iron-ajax` ? (similar to how we have `on-response` on `iron-ajax`)

https://www.webcomponents.org/element/PolymerElements/iron-ajax/v2.1.3/elements/iron-ajax#event-error

**Comment by 13guff13** [comment]
> @13guff13
> 
> why not use `on-error` handler on `iron-ajax` ? (similar to how we have `on-response` on `iron-ajax`)
> 
> https://www.webcomponents.org/element/PolymerElements/iron-ajax/v2.1.3/elements/iron-ajax#event-error

<iron-ajax error="errorhandler" - doesn't work so I just try another method

**Comment by jkanche** [comment]
> > @13guff13
> > why not use `on-error` handler on `iron-ajax` ? (similar to how we have `on-response` on `iron-ajax`)
> > https://www.webcomponents.org/element/PolymerElements/iron-ajax/v2.1.3/elements/iron-ajax#event-error
> 
> <iron-ajax error="errorhandler" - doesn't work so I just try another method

it should be 

```
<iron-ajax on-error="errorhandler" ...
```

File: epiviz-mbrowser.html [hunk 1]
```diff
@@ -156,7 +156,6 @@
       }
     </style>
     <!-- local DOM goes here -->
-
     <iron-ajax id="getProjects" method="GET" 
       url="[[msApi]]projects/"
       handle-as="json" on-response="handleGetProjects" debounce-duration="300">
```
Comment (jkanche) [inline epiviz-mbrowser.html:161]: use `on-error` on `iron-ajax` to capture errors

**Review by jkanche — CHANGES_REQUESTED** [review]

File: epiviz-mbrowser.html (hunk 1 above)
Comment (13guff13) [reply epiviz-mbrowser.html:162]: done

File: epiviz-mbrowser.html [hunk 2]
```diff
@@ -951,7 +947,6 @@
         var project = collection.split("::");
 
         const request = this.$.getProjects.generateRequest();
```
Comment (jkanche) [inline epiviz-mbrowser.html:949]: declaring `const request` is also unnecessary

**Review by jkanche — CHANGES_REQUESTED** [review]

File: epiviz-mbrowser.html [hunk 3]
```diff
@@ -283,6 +302,11 @@
             value: "hg38"
           },
 
+          error: {
+            type: String,
```
Comment (jkanche) [inline epiviz-mbrowser.html:306]: don't you want this to be a `Boolean` instead if `string` ?

File: epiviz-mbrowser.html [hunk 4]
```diff
@@ -454,8 +478,19 @@
         super.ready();
       }
 
+      isError(error) {
+        return !!error
+      }
+      showContent(error) {
+        return !error
+      }
```
Comment (jkanche) [inline epiviz-mbrowser.html:486]: using `error` as type Boolean would make this redundant

**Review by jkanche — CHANGES_REQUESTED** [review]

File: epiviz-mbrowser.html (hunk 3 above)
Comment (jkanche) [reply epiviz-mbrowser.html:306]: is there a reason why you want this to be a string ?

File: epiviz-mbrowser.html (hunk 4 above)
Comment (13guff13) [reply epiviz-mbrowser.html:486]: agree, will change it

**Pull request closed without merging**